### ISSUE TITLE
forward_t for BatchNorm printing wrong value?

### ISSUE THREAD

**Issue by danjavady** [LaurentMazare/tch-rs#544]
https://docs.rs/tch/latest/src/tch/nn/batch_norm.rs.html#92-111

at line 95 why not print the `dim` and `nd` over `size()`?  I am new to rust so I am not sure if im missing something here.

I ran into this error message 
```thread 'main' panicked at 'expected an input tensor with 2 or 3 dims, got [2]```
and was wondering if this was intended.

**Comment by LaurentMazare** [comment]
I've pushed some small changes to make the error message slightly more explicit (`dim` is just the length of `size()` so `size()` gives more information and could be useful to locate the actually problematic tensor but mentioning both sounds reasonable).
